examples/python/DynamicCalibration/calibration_dynamic.py

```
@@ -63,23 +58,13 @@
-        npDisparity = disparity.getFrame()
-        curMax = float(np.max(npDisparity))
-        if curMax > 0:
-            maxDisparity = max(maxDisparity, curMax)
-
-        # Normalize to [0,255] and colorize; keep zero-disparity as black
-        denom = maxDisparity if maxDisparity > 0 else 1.0
-        normalized = (npDisparity / denom * 255.0).astype(np.uint8)
-        colorizedDisparity = cv2.applyColorMap(normalized, cv2.COLORMAP_JET)
-        colorizedDisparity[normalized == 0] = (0, 0, 0)
-        cv2.imshow("disparity", colorizedDisparity)
+        colorizedDepth = dai.colorizeDepthFrame(depth, 500, 12000, cv2.COLORMAP_JET, True).getCvFrame()
```

Ruff (0.15.21)

[warning] Boolean positional value in function call

(FBT003)